test/source/tests/gmail.ts

```diff
@@ -289,11 +311,12 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       )
     );
 
-    test(
+    // convo-sensitive, draft-sensitive test
+    test.serial(
```

Comment:
It'll be run serially anyway. I can leave this `serial` as precaution or delete it